config/webpacker.yml

```diff
@@ -77,7 +77,11 @@ development:
 
```

Comment:
Reminder: You'll want to delete my comment above `extract_css: false` in the default section if we're keeping that value `false`.